DigiMeSDK/Core/Classes/Utility/DMECompressor.m

```diff
@@ -0,0 +1,61 @@
+//
+//  DMECompressor.m
+//  DigiMeSDK
+//
+//  Created on 21/11/2018.
+//
+
+#import "DMECompressor.h"
+@import Brotli;
+
+@implementation DMECompressor
+
++ (nullable NSData *)compressData:(NSData *)data usingAlgorithm:(DMECompressionAlgorithm)algorithm
+{
+    switch (algorithm)
+    {
+        case 0:
```

Comment:
Would prefer to use the enum names here, ie `DMECompressionAlgorithmGZIP` / `DMECompressionAlgorithmBrotli`